Client/src/components/game/leftBarSmall.tsx

```diff
@@ -0,0 +1,205 @@
+import '../css/leftBarSmall.css';
+import { useState } from 'react';
+import { minionType, TowerType } from '../../utils/types';
+import FlagSVG from '../svg/flagSVG';
+import { useAppSelector } from '../../features/hooks';
+import { ZoomInSVG, ZoomOutSVG } from '../svg/zoomButtonsSVG';
+import TowerSVG from '../svg/towerSVG';
+import { zoomHover, whichAnimalSVG } from './leftBar'; // zoomIn, zoomOut now dispatch
+import MediaQuery from 'react-responsive';
+
+export default function LeftBarSmall({
+  currentMinion,
+  minions,
+  currentTower,
+  gameStats,
+  towers,
+}: {
+  currentMinion: null | number;
+  currentTower: null | TowerType;
+  minions: { [key: number]: minionType };
+  gameStats: {
+    timeRemaining: number;
+    p1Coins: number;
+    p2Coins: number;
+    p1Towers: number[];
+    p2Towers: number[];
+    p1MinionCount: number;
+    p2MinionCount: number;
+  };
+  towers: TowerType[];
+}) {
+  const [toggle, setToggle] = useState(false);
+
+  const user = useAppSelector((state) => state.user);
+
+  const toggleLeftBar = () => {
+    setToggle(!toggle);
+  };
+
+  return (
+    <>
+      <MediaQuery maxWidth={950}>
+        <div
+          className="container-left-bar"
+          onClick={() => {
+            toggleLeftBar();
+          }}
+        >
+          <div className={`plus ${toggle}`}>+</div>
+          <div className={`minus ${toggle}`}>-</div>
+        </div>
+      </MediaQuery>
```

Comment:
I understand this is a work in progress push

Reply:
yes